DOCS-5938 add examples to JS /organization/invitations reference (#824)
clerk/clerk-docs: docs/references/javascript/organization/invitations.mdx

---
title: Organization invitation methods

> description: Learn about the methods on the Organization object that allow you to manage the invitations to an organization.

Reviewer: ```suggestion
description: Learn how to manage the invitations to an organization.
```
I know I haven't brought this up on other PRs, but what do you think of shortening these descriptions? Especially since we're adding "methods" to the title I think the shorter version is clear enough

Author: I think "Learn how to manage the invitations to an organization" sounds like it's going to be a guide, and the first description makes it pretty clear that its a reference document that is going to include information about methods.

| `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
| `status?` | `'pending' \| 'accepted' \| 'revoked'` | The status an invitation can have. |

### `getInvitations()` returns

| Type | Description |
| --- | --- |
| `Promise<ClerkPaginatedResponse<OrganizationInvitation>>` | This method returns a `Promise` that resolves to a [`ClerkPaginatedResponse`](/docs/references/javascript/types/clerk-paginated-response) of [`OrganizationInvitation`][orginv-ref] objects. |

### `getInvitations()` example

<InjectKeys>

<CodeBlockTabs type="npm-script" options={['NPM module', '<script>']}>
```js filename="main.js"
import Clerk from '@clerk/clerk-js';

// Initialize Clerk with your Clerk publishable key
const clerk = new Clerk('{{pub_key}}');
await clerk.load();

if (clerk.user) {

>   await clerk.organization.getInvitations()

Reviewer: Also have to check if clerk.organization != null in these examples too

| `limit` | `number` | Limit of the results returned. |
| `offset` | `number` | The offset of results to start the retrieval. |

> ### `Returns`

Reviewer: ```suggestion
### `getPendingInvitations` returns
```
Should be this, no?

Author: its completely removed in the beta docs so i didn't even want to touch this section, but i can update it for consistency in the core 1 docs, thats fine
